api/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/', (req, res) => {
+    res.status(200).json('the line is hot');
+});
```

Comment:
Nice and easy GET endpoint to verify that data is successfully posted and deleted.